Packs/FortiGate/ReleaseNotes/2_0_2.md

```diff
@@ -0,0 +1,141 @@
+
+#### Modeling Rules
+
+##### Fortinet FortiGate Modeling Rule
+
+- Updated XDM mappings for the following fields: 
+| CEF Log Field Name      | Fortigate Field Name | Field Description     | XDM Target Field| 
+| :---        | :----       | :---          | :---       | 
+`FTNTFGTapp`|app|Application Name|`xdm.target.application.name` ~~`xdm.source.application.name`~~|
+`FTNTFGTappcat` ~~FTNTFGTcat~~|appcat|Application Category|`xdm.network.application_protocol_category`|
+`FTNTFGTdevtype` ~~FTNTFGTsrcfamily~~|devtype|Device type of the source|`xdm.source.host.device_category`|
+
+- Added XDM mappings for the following fields: 
+| CEF Log Field Name      | Fortigate Field Name | Field Description     | XDM Target Field| 
+| :---        | :----       | :---          | :---       | 
+`c6a2`|c6a2|Source IPv6 Address|`xdm.source.ipv6`|
```

Comment:
Here too.  Do we really need all this information in the release notes?

Reply:
Updated to be more concise.